taxi/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+
+# Register your models here.
+from django.contrib import admin

```

Comment:
The import statement for `django.contrib.admin` is duplicated. You can remove the second import on line 4 to avoid redundancy.